types/unist/unist-tests.ts

```diff
@@ -1,56 +1,93 @@
-import { Data, Point, Position, Node, Literal, Parent } from 'unist';
+import { Point, Position, Node, Literal, Parent, NodeData } from 'unist';
 
-const data: Data = {
+const data = {
     string: 'string',
     number: 1,
     object: {
-        key: 'value'
+        key: 'value',
     },
     array: [],
     boolean: true,
-    null: null
+    null: null,
 };
 
 const point: Point = {
     line: 1,
     column: 1,
-    offset: 0
+    offset: 0,
 };
 
 const position: Position = {
     start: point,
     end: point,
-    indent: [1]
+    indent: [1],
 };
 
 const node: Node = {
     type: 'node',
     data,
-    position
+    position,
 };
 
-const text: Literal = {
+const text: Literal<string> = {
     type: 'text',
     data,
     position,
-    value: 'value'
+    value: 'value',
 };
 
 const parent: Parent = {
     type: 'parent',
     data,
     position,
-    children: [node, text]
+    children: [node, text],
 };
 
 const noExtraKeysInNode: Node = {
     type: 'noExtraKeysInNode',
     // $ExpectError
-    extra: 'extra'
+    extra: 'extra',
 };
 
 const noChildrenInNode: Node = {
     type: 'noChildrenInNode',
     // $ExpectError
-    children: []
+    children: [],
+};
+
+const literalParent: Parent<Literal<string>> = {
+    type: 'literalParent',
+    data,
```

Comment:
what happens when parent has different data than children?
Also could an example with nested parents be added?

Reply:
There is.

```ts
const literalParentData: Parent<Literal<string>, typeof data>
```